cdc/sink/producer/kafka/config.go

```diff
@@ -72,7 +72,7 @@ func (c *Config) CompleteByOpts(sinkURI *url.URL, replicaConfig *config.ReplicaC
 	params := sinkURI.Query()
 	s := params.Get("partition-num")
 	if s != "" {
-		a, err := strconv.Atoi(s)
```

Comment:
why this is a security problem :joy:

Reply:
becasue  in L79 `c.PartitionNum = int32(a)` try convert int to int32 😂